modules/flowable-engine/src/test/java/org/flowable/engine/test/api/event/DatabaseEventLoggerTest.java

```diff
@@ -97,193 +99,201 @@ public void testDatabaseEvents() throws IOException {
-                assertNotNull(data.get(Fields.PROCESS_DEFINITION_ID));
-                assertNotNull(data.get(Fields.PROCESS_INSTANCE_ID));
-                assertNotNull(data.get(Fields.EXECUTION_ID));
-                assertNotNull(data.get(Fields.ACTIVITY_TYPE));
-                assertEquals(testTenant, data.get(Fields.TENANT_ID));
+                assertThat(data.get(Fields.ACTIVITY_ID)).isNotNull();
+                assertThat(data.get(Fields.PROCESS_DEFINITION_ID)).isNotNull();
+                assertThat(data.get(Fields.PROCESS_INSTANCE_ID)).isNotNull();
+                assertThat(data.get(Fields.EXECUTION_ID)).isNotNull();
+                assertThat(data.get(Fields.ACTIVITY_TYPE)).isNotNull();
+                assertThat(data.get(Fields.TENANT_ID)).isEqualTo(testTenant);
 
             }
 
             // Tasks
             if (i == 10 || i == 14) {
 
-                assertNotNull(entry.getType());
-                assertEquals(FlowableEngineEventType.TASK_CREATED.name(), entry.getType());
-                assertNotNull(entry.getTimeStamp());
-                assertNotNull(entry.getProcessDefinitionId());
-                assertNotNull(entry.getProcessInstanceId());
-                assertNotNull(entry.getExecutionId());
-                assertNotNull(entry.getTaskId());
+                assertThat(entry.getType()).isNotNull();
+                assertThat(entry.getType()).isEqualTo(FlowableEngineEventType.TASK_CREATED.name());
+                assertThat(entry.getTimeStamp()).isNotNull();
+                assertThat(entry.getProcessDefinitionId()).isNotNull();
+                assertThat(entry.getProcessInstanceId()).isNotNull();
+                assertThat(entry.getExecutionId()).isNotNull();
+                assertThat(entry.getTaskId()).isNotNull();
 
                 Map<String, Object> data = objectMapper.readValue(entry.getData(), new TypeReference<HashMap<String, Object>>() {
+
                 });
-                assertNotNull(data.get(Fields.ID));
-                assertNotNull(data.get(Fields.NAME));
-                assertNotNull(data.get(Fields.ASSIGNEE));
-                assertNotNull(data.get(Fields.CREATE_TIME));
-                assertNotNull(data.get(Fields.PRIORITY));
-                assertNotNull(data.get(Fields.PROCESS_DEFINITION_ID));
-                assertNotNull(data.get(Fields.EXECUTION_ID));
-                assertNotNull(data.get(Fields.TENANT_ID));
-
-                assertFalse(data.containsKey(Fields.DESCRIPTION));
-                assertFalse(data.containsKey(Fields.CATEGORY));
-                assertFalse(data.containsKey(Fields.OWNER));
-                assertFalse(data.containsKey(Fields.DUE_DATE));
-                assertFalse(data.containsKey(Fields.FORM_KEY));
-                assertFalse(data.containsKey(Fields.USER_ID));
-
-                assertEquals(testTenant, data.get(Fields.TENANT_ID));
+                assertThat(data.get(Fields.ID)).isNotNull();
```

Comment:
For maps we can in theory use `assertThat(data).containsOnlyKeys(...)`.

Reply:
@filiphr Made the changes per your note and also fixed the ordering of `.as()`.  Thanks